src/transformers/babelSyntax.js

```diff
@@ -11,15 +11,19 @@ function cleanStackTrace(message) {
 
 function cleanMessage(message) {
   return message
-  // match until the last semicolon followed by a space
-  // this should match
-  // linux => "(SyntaxError: )Unexpected token (5:11)"
-  // windows => "(SyntaxError: C:/projects/index.js: )Unexpected token (5:11)"
-    .replace(/^Module build failed.*:\s/, 'Syntax Error: ');
+    // match until the last semicolon followed by a space
+    // this should match
+    // linux => "(SyntaxError: )Unexpected token (5:11)"
+    // windows => "(SyntaxError: C:/projects/index.js: )Unexpected token (5:11)"
+    .replace(/^Module build failed.*:\s/, 'Syntax Error: ')
+    // remove mini-css-extract-plugin loader tracing errors
+    .replace(/^Syntax Error: ModuleBuildError:.*:\s/, '')
+    // remove babel extra wording
+    .replace('SyntaxError: ', '');
 }
 
 function isBabelSyntaxError(e) {
-  return e.name === 'ModuleBuildError' &&
+  return e.name === 'ModuleBuildError' || e.name === 'ModuleBuildError' &&
```

Comment:
are you testing the same string ?

Reply:
`e.name === 'ModuleBuildError' && e.message.indexOf('SyntaxError') >= 0;` this one only works both exist wich casue on SCSS errors can't catch it.

So I am adding if no `e.message` which still can catch `'ModuleBuildError'`.